python/custom_model/model.py

```diff
@@ -99,8 +102,19 @@ def preprocess(
         return input_tensor.unsqueeze(0)
 
     def predict(
-        self, input_tensor: torch.Tensor, headers: Dict[str, str] = None
+        self,
+        input_tensor: torch.Tensor,
+        headers: Dict[str, str] = None,
+        response_headers: Dict[str, str] = None,
     ) -> Union[Dict, InferResponse]:
+
+        global PREDICTION_COUNTER
+        PREDICTION_COUNTER += 1
```

Comment:
global variable is not an good example, we can use predict time latency as the example response header.